src/styles/styles.js

```diff
@@ -636,6 +636,35 @@ const styles = {
         marginLeft: 48,
     },
 
+    chatItemSingleWrapper: {
+        display: 'flex',
+        flexDirection: 'row',
+        padding: 0,
+    },
+
+    chatItemSingleAvatarWrapper: {
+        position: 'absolute',
+        zIndex: 1,
+        top: 8,
+        left: 20,
+    },
+
+    chatItemHeaderAbsolute: {
+        paddingLeft: 20,
+        paddingRight: 20,
```

Comment:
@mollfpr could you please help me understand why padding is necessary?
Wouldn't margin be more appropriate here?

<img width="359" alt="Screen Shot 2022-04-07 at 1 19 39 PM" src="https://user-images.githubusercontent.com/29673073/162152167-f2d075d2-e34d-4e91-b6d4-60dfb71f2db1.png">
 

<img width="524" alt="image" src="https://user-images.githubusercontent.com/29673073/162152967-f56eaf6c-d406-470e-a183-826f6e45a3a5.png">

Reply:
That’s padding use for the name and timestamp not close to the edge on small device.

Reply:
Actually only right side is needed but I just follow the last style 😅

Reply:
We broke elipsis for long names. Can we think of an alternative approach?

https://user-images.githubusercontent.com/29673073/162156201-eee1ce84-71f8-4270-b19c-ebec53c57c02.mov

Reply:
Interesting. I’ll take a look on that. Thanks!

Reply:
Thanks for your patience!

Reply:
I update the styles. 

I remove the `left` and add the value to `paddingLeft` and add `width: 100%` to maintain the width.

---

<img width="447" alt="Screen Shot 2022-04-07 at 22 03 44" src="https://user-images.githubusercontent.com/25520267/162231701-8f467623-2a5f-4c5c-827a-49ea45c531f1.png">

https://user-images.githubusercontent.com/25520267/162235915-f4e12ac3-6328-4e19-afd8-84b2491a0f20.mov